lib/galaxy/util/sanitize_html.py

```diff
@@ -112,17 +114,17 @@ def unknown_starttag(self, tag, attrs):
                 value = value.replace('>', '&gt;').replace('<', '&lt;').replace('"', '&quot;')
                 value = self.bare_ampersand.sub("&amp;", value)
                 # thanks to Kevin Marks for this breathtaking hack to deal with (valid) high-bit attribute values in UTF-8 feeds
-                if isinstance(value, unicode):
+                if isinstance(value, text_type):
                     try:
-                        value = unicode(value, self.encoding)
-                    except:
-                        value = unicode(value, 'iso-8859-1')
-                uattrs.append((unicode(key, self.encoding), value))
+                        value = unicodify(value, self.encoding)
+                    except Exception:
+                        value = unicodify(value, 'iso-8859-1')
```

Comment:
> @mvdbeek Are you using feedparser from pip (i.e. 5.2.1) or from the develop branch on GitHub?

I started with pip, then tried the github develop branch.


> What was the problem with the html module? It should be fixable without a rename.

It was probably me being in the feedparser directory, trying to import html.entities from the html module, not from html.py ... .

Indeed, I'll create an issue for feedparser/sgmllib in #1715